### PR TITLE
Emitting afterAddTagsToProducts, afterTagDelete, afterRemoveTagsFromProducts

### PR DESCRIPTION
**Pull request by vishalmalu**
Impact: minor
Type: feature

**Issue**
This PR emits following additional events when respective mutations have been executed.

- afterAddTagsToProducts
- afterRemoveTagsFromProducts
- afterTagDelete

### CODE REVIEW

**Review by tedraykov — CHANGES_REQUESTED** [review]
Please run `changeset` on `npx changeset` to generate the changelog files. This is required to bump the plugin versions.
Apart from that the PR is good to go.

**Comment by vishalmalu** [comment]
@tedraykov I've added changeset.

File: packages/api-plugin-products/src/mutations/addTagsToProducts.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export default async function addTagsToProducts(context, input) {
   const { productIds, shopId, tagIds } = input;
   const { collections: { Products } } = context;
   const totalProducts = productIds.length;
+  const { appEvents } = context;
```
Comment (brent-hoover) [inline packages/api-plugin-products/src/mutations/addTagsToProducts.js]: You are already deconstructing context on line 16, should probably just get `appEvents` from there.

File: packages/api-plugin-products/src/mutations/removeTagsFromProducts.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ export default async function removeTagsFromProducts(context, input) {
   const { productIds, shopId, tagIds } = input;
   const { collections: { Products } } = context;
   const totalProducts = productIds.length;
+  const { appEvents } = context;
```
Comment (brent-hoover) [inline packages/api-plugin-products/src/mutations/removeTagsFromProducts.js]: Same comment re: context as above, also everywhere else you are getting appEvents

**Review by brent-hoover — CHANGES_REQUESTED** [review]

**Review by tedraykov — APPROVED** [review]

**Review by brent-hoover — APPROVED** [review]

**Pull request merged**